test/absinthe/execution/subscription_test.exs

```diff
@@ -49,7 +49,7 @@ defmodule Absinthe.Execution.SubscriptionTest do
     @behaviour Absinthe.Subscription.Pubsub
 
     def start_link() do
-      Registry.start_link(keys: :unique, name: __MODULE__)
+      Registry.start_link(keys: :duplicate, name: __MODULE__)
```

Comment:
Had to change this to allow duplicated keys to make the test fail and also because the registry in the sub supervisor allows duplicated keys: https://github.com/absinthe-graphql/absinthe/blob/master/lib/absinthe/subscription/supervisor.ex#L37

Reply:
Yup, good call.